torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -3293,11 +3293,19 @@ Error PyTorchModelLoader::loadConvolution(const torch::jit::Node *ptNode) {
                         input.dims().size() == weights.dims().size(),
                     "Expect 4 dims in input and weights for conv2d and 5 dims "
                     "in input and weights for conv3d");
+
+  bool transposed;
+  ASSIGN_VALUE_OR_RETURN_ERR(transposed, iValToBool(getGlowIValueForValue(
+                                             inputs[ConvInputs::transposed])));
+
   bool isConv3d = input.dims().size() == 5;
   if (isConv3d) {
     input = F_.createTranspose("conv_input_transposed", input, NCTHW2NTHWC);
     weights =
         F_.createTranspose("conv_weights_transposed", weights, NCTHW2NTHWC);
+  } else if (transposed) {
+    input = F_.createTranspose("conv_input_transposed", input, NCHW2NHWC);
+    weights = F_.createTranspose("conv_weights_transposed", weights, CNHW2NHWC);
```

Comment:
Why this?

Reply:
Hi @zrphercule if you are asking why the weights are transposed -- in the [onnx definition of conv transpose](https://github.com/onnx/onnx/blob/master/docs/Operators.md#ConvTranspose), the weights are in cnhw but the expected format is nhwc for glow. Since we are mapping to existing glow node, this is needed.